pre_commit/xargs.py

```diff
@@ -170,7 +170,8 @@ def run_cmd_partition(
         results = thread_map(run_cmd_partition, partitions)
 
         for proc_retcode, proc_out, _ in results:
-            retcode = max(retcode, proc_retcode)
+            if abs(proc_retcode) > abs(retcode):
+                retcode = proc_retcode
```

Comment:
I used this over something like `retcode = retcode or proc_retcode` just so that it would consistently pick the same exit status from a set of exit statuses. If you prefer some other way of selecting which to use, I can change it to that.